src/components/Chart.js

```diff
@@ -25,81 +26,85 @@ const tenors = [
+        {selectedChartData.length !== 0
+          ? tenors.map(({ value, label }) => {
               const activeClass = active === value ? "--active" : "";
 
               return (
                 <button
+                  key={label}
                   className={`chart__button chart__button${activeClass}`}
                   onClick={onClickHandler}
                   value={value}
                 >
                   {label}
                 </button>
               );
-            })}
-          </div>
-          <ResponsiveContainer
-            height="100%"
-            width="100%"
-            minHeight="350px"
-            maxHeight="400px"
-          >
-            <AreaChart
-              data={chartData}
-              margin={{ top: 10, right: 0, left: 0, bottom: 0 }}
-            >
-              <defs>
-                <linearGradient id="chartGradient" x1="0" y1="0" x2="0" y2="1">
-                  <stop offset="5%" stopColor="#8884d8" stopOpacity={0.8} />
-                  <stop offset="95%" stopColor="#8884d8" stopOpacity={0} />
-                </linearGradient>
-              </defs>
-              <CartesianGrid strokeDasharray="3 3" />
-              <XAxis dataKey="date" valueFormatString="MMM" />
-              <YAxis orientation="right" />
-              <Tooltip />
-              <ReferenceLine
-                y={stockTickerLatestPrice.latestPrice}
-                label={{
-                  value: `${stockTickerLatestPrice.latestPrice}`,
-                  position: "right",
-                  fill: "orange"
-                }}
-                stroke="orange"
-                strokeDasharray="3 3"
-              />
-              <Area
-                type="monotone"
-                dataKey="close"
-                stroke="#8884d8"
-                fillOpacity={1}
-                fill="url(#chartGradient)"
-                connectNulls={true}
-              />
-            </AreaChart>
-          </ResponsiveContainer>
-        </>
-      )}
+            })
+          : "Chart data N/A"}
+      </div>
+      <ResponsiveContainer
+        height="100%"
+        width="100%"
+        minHeight="350px"
+        maxHeight="400px"
+      >
+        <AreaChart
+          data={selectedChartData}
+          margin={{ top: 10, right: 0, left: 0, bottom: 0 }}
+        >
+          <defs>
+            <linearGradient id="chartGradient" x1="0" y1="0" x2="0" y2="1">
+              <stop offset="5%" stopColor="#8884d8" stopOpacity={0.8} />
+              <stop offset="95%" stopColor="#8884d8" stopOpacity={0} />
+            </linearGradient>
+          </defs>
+          <CartesianGrid strokeDasharray="3 3" />
+          <XAxis dataKey="date" valueFormatString="MMM" />
+          <YAxis orientation="right" />
+          <Tooltip />
+          <ReferenceLine
+            y={selectedStockTicker.latestPrice}
+            label={{
+              value: `${selectedStockTicker.latestPrice}`,
```

Comment:
Instead of a string template, you could just use the String constructor-function:
`String(selectedStockTicket.latestPrice)`
Or, preferably, just pass in the number (if that works)